frontend/tests/utils/mocks/ethers.ts

```diff
@@ -12,7 +12,13 @@ export class MockedTransaction {
 }
 export class MockedTransactionReceipt {
   logs = [];
+  status = 1;
```

Comment:
We can create a constructor here and set the value of the status based on the value of the passed argument to the constructor. 
Default will be 1.

Then we can use this mock class for both cases.